src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -27,13 +26,14 @@
 ACR_CHECK_HEALTH_MSG = "Try running 'az acr check-health -n {} --yes' to diagnose this issue."
 RECOMMENDED_NOTARY_VERSION = "0.6.0"
 NOTARY_VERSION_REGEX = re.compile(r'Version:\s+([.\d]+)')
+DOCKER_PULL_WRONG_PLATFORM = 'cannot be used on this platform'
 
 
 # Utilities functions
 def print_pass(message):
     from colorama import Fore, Style, init
     init()
-    print(str(message) + " : " + Fore.GREEN + "OK" + Style.RESET_ALL, file=sys.stderr)
+    logger.warning(str(message) + " : " + Fore.GREEN + "OK" + Style.RESET_ALL)
```

Comment:
pls be aware of that warning will display to stderr. Some customers who're doing automation don't want warning in stderr to break auto pipeline

Reply:
and we don't suggest color in output, except it's from CLI underlying framework.

Reply:
Removed